alpine/content.md

```diff
@@ -0,0 +1,30 @@
+# What is Alpine Linux?
+
+[Alpine Linux](http://alpinelinux.org/) is a Linux distribution built around [musl libc](http://www.musl-libc.org/) and [BusyBox](http://www.busybox.net/). The image is only 5 MB in size and has access to a [package repository](http://forum.alpinelinux.org/packages) that is much more complete than other BusyBox based images. This makes Alpine Linux a great image base for utilities and even production applications. [Read more about Alpine Linux here](https://www.alpinelinux.org/about/) and you can see how their mantra fits in right at home with Docker images.
+
+# How to use this image
+
+## Usage
+
+Stop doing this:
+
+	FROM ubuntu:14.04
+	RUN apt-get update -q \
+	  && DEBIAN_FRONTEND=noninteractive apt-get install -qy mysql-client \
+	  && apt-get clean \
+	  && rm -rf /var/lib/apt
+	ENTRYPOINT ["mysql"]
+
+This took 19 seconds to build and yields a 164 MB image. Eww.
```

Comment:
I'm not sure where this size comes from -- is this supposed to be the size of the additional layers?  The size of the total image?

Also, I'm not sure where "19 seconds" comes from either.  Is that with a primed apt-cacher-ng instance?

I'm not opposed to the highly-subjective "Eww", but the facts leading to it aren't exactly accurate, and I think more accurate (or at least actually verifiable) facts will add even more punch to the "Eww". :+1: